lib/active_encode/engine_adapters/test_adapter.rb

```diff
@@ -23,18 +23,18 @@ def find(id, _opts = {})
         new_encode
       end
 
```

Comment:
now you call merge in the Core.find, do we still need dup here?

Reply:
Good catch.  I'll clean this up.